### DISCUSSION TITLE
`Ookii.Dialogs.WinForms` is not integrated with the Form Designer

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#2942] [General]
`Ookii.Dialogs.WinForms` controls are not integrated. An entry for `Ookii.Dialogs.WinForms` is added but there is no controls available other than `Pointer`. Can't drag and drop the controls into the form so using of them relying entirely on manually coding.

**Comment by SunSerega** [comment]
It's weird that any control shows up in there. It wasn't designed to handle anything other than WinForms.

**Reply by ghost** [reply]
Disclaimer: I have never used `Ookii.Dialogs.WinForms` with C# or the PascalABC.NET Form Designer, but from what I have read from `Ookii.Dialogs.WinForms`, it seems `Ookii.Dialogs.WinForms` is integrated into Visual Studio's Form Designer:

https://github.com/ookii-dialogs/ookii-dialogs-winforms/blob/master/sample/shared/MainForm.Designer.cs

**Reply by SunSerega** [reply]
Well, and I'm always defining controls in code. Sometimes with XAML. Classical editors are only good for throwing stuff together, but actually making it look good is hard. So your guess is as good as mine.

**Reply by ghost** [reply]
> It wasn't designed to handle anything other than WinForms.

`Ookii.Dialogs.WinForms` is a collection of controls for WinForms.
